subsys/bluetooth/services/fast_pair/fmdn/Kconfig

```diff
@@ -26,6 +26,10 @@ config BT_FAST_PAIR_FMDN_TX_POWER
 	  procedure of this library. The actual TX power value will be as close to
 	  the desired configuration as possible.
 
+	  This Kconfig option must be set to zero at minimum. The Fast Pair specification
+	  requires that the conducted Bluetooth transmit power for FMDN advertisements
+	  must be set to at least 0 dBm.
```

Comment:
just use a `range 0 <max>` instead?

Reply:
there is no clear max value for this option

(BT_FAST_PAIR_FMDN_TX_POWER + BT_FAST_PAIR_FMDN_TX_POWER_CORRECTION_VAL) <= 20

Reply:
done